docs/geedocs/answer/4412455.html

```diff
@@ -145,6 +145,25 @@
     <h2><a name="imagery_proj"></a>Define a Flat Imagery Project</h2>
     <p>The maps layer includes Google Maps layers and uses the Google Maps API from google.com. Map databases use either Plate Carrée (EPSG: 4326) and Mercator (EPSG: 3857) projections and normally require corresponding imagery resource types. In this lesson you add a flat imagery project from a previous tutorial lesson to a Mercator map database, which can then be projected as <em>Mercator on the Fly</em>. For more information, see <a href="../answer/6081069.html">Add flat imagery to Mercator map databases</a>.</p>
     <div class="tip">If you did not complete the tutorial lesson to create an imagery project, see <a href="../answer/4412421.html#56899">Define an Imagery Project</a>. </div>
+    <ol class="spaced-list">
+      <li>Select <b>Asset Manager</b> from the Tools menu. The Asset Manager appears.</li>
+      <li>Select <b>Mercator Map Tools (2D)</b>. The tools for Mercator maps appear in the toolbar.</li>
+     <li>Click <img src="../art/fusion/tutorial/iconResImageryMerc.png"/>.  The Mercator Imagery Resource Editor appears with no resources selected.<br/>
+      <img src="../art/fusion/tutorial/imageryResEditorMerc.png"/></li>
+     <li>Click <strong>Add</strong><br/>
+      The Open Source dialog opens to the <code>/opt/google/share/tutorials/fusion/Imagery</code> folder.<br/>
+      <img src="../art/fusion/tutorial/imageryOpen-2.png"/></li>
+     <li>Select <strong>usgsLanSat.tif</strong>, and click <strong>Open</strong>.</li>
```

Comment:
I'm running into problems with this.  When I follow these instructions I end up with a map that only has imagery around the bay area.  There's a couple of ways to fix this that I can think of:

- I'm pretty sure it's possible to specify a non-mercator imagery project when building a mercator database.  If that works fine we can just use the SFBayArea project and not worry about building an SFBayAreaMercator project.
- The other option would be to build the SFBayAreaMercator project as a complete copy of the SFBayArea project including all the same resources.